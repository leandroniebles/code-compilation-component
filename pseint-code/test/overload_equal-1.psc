Proceso PruebaIgual
  a=0;
  Para i=1 hasta 10 hacer
    Escribir i;
  FinPara
FinProceso