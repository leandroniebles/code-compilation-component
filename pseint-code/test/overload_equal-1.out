=== Line 2: SynError 106
=== Line 3: SynError 72
